### DISCUSSION TITLE
revealjs: format of date entry on title page

### DISCUSSION THREAD

**Discussion by mtthsdzwn** [quarto-dev/quarto-cli#3265] [Q&A] [answered]
I am making a revealjs presentation. I wrote the date entry in the YAML as day month year (eg 10 November 2022). But on the title page, the date is formatted as `11/10/22`. How do I change that format to `10 November 2022`?

**Comment by dragonstyle** [comment, marked as answer]
Quarto supports formatting the date - you can control the format using `date-format`- more about that here:

https://quarto.org/docs/reference/dates.html

**Comment by pstoyanov** [comment]
While I don't see it explicitly mentioned in the docs -- apparently you can provide any text string as "date format". If it does not match the date formats, it gets printed as-is. In my case -- supplying `date-format: 2022/2023` does the job to show academic year. Note that you still need some value for date, otherwise it is not "formatted".

Thanks for the useful tool!

**Comment by elcansaito** [comment]
I have the issue in another way. I want to use the R system date to include it in the YAML , but it doesn´t work. 
For example. 

---
title: "Test date"
author:  JIS
date: "Last compiled on `r format(Sys.time(), '%d %B, %Y')`"
title-slide-attributes: 
  data-background-image: "https://raw.githubusercontent.com/elcansaito/images/main/Background.png"
  #data-background-opacity: "0.5"
format: 
  revealjs:
    self-contained: true
    slide-number: true
    width: 1600
    height: 900
    css: ["theme.css"]
    theme: simple
editor: source
execute: 
  echo: true
editor_options: 
  chunk_output_type: inline

---

This doesn´t work even though I change it as   date: "`r format(Sys.Date(), '%Y-%b-%d')`".

The only way to work is if I use something like this.

date: 2023/10/11
date-format: "YYYY/MMMM/DD"

However, I´d like R to run it for me every time. Any solution?
THANKS A MILLION

**Comment by mcanouil** [comment]
Please open a new discussion and format properly your example by enclosing it in a code block with more backticks (four).

**Reply by elcansaito** [reply]
Thanks mcanouil for the feedback. I created a new one using the backticks.
